README.md

```diff
@@ -155,38 +167,124 @@ obj->extensionMethod();
 // Compare with extensionMethod.call(obj).
 ```
 
-The bind operator can also **extract** a **method** from a **class**
-into a function whose first parameter becomes its `this` binding:\
-for example, `const { slice } = Array.prototype; arr->slice(1, 3);`.\
-It can also similarly extract a method from an **instance**
-into a function that always uses that instance as its `this` binding:\
-for example, `const arr = arr->(arr.slice); slice(1, 3);`.
-
 ## Real-world examples
 Only minor formatting changes have been made to the status-quo examples.
 
-<table>
-<thead>
-<tr>
-<th>Status quo
-<th>With binding
+### Node.js
+Node.js’s runtime depends on many built-in JavaScript global intrinsic objects
+that are vulnerable to mutation or prototype pollution by third-party libraries.
+When initializing a JavaScript runtime, Node.js therefore caches
+wrapped versions of all global intrinsic objects (and their methods)
+in a [large `primordials` object][primordials.js].
+This `primordials` object contains numerous entries that look like this:
+```js
+ArrayPrototypeConcat: uncurryThis(Array.prototype.concat),
+ArrayPrototypeCopyWithin: uncurryThis(Array.prototype.copyWithin),
+ArrayPrototypeFill: uncurryThis(Array.prototype.fill),
+ArrayPrototypeFind: uncurryThis(Array.prototype.find),
+ArrayPrototypeFindIndex: uncurryThis(Array.prototype.findIndex),
+ArrayPrototypeLastIndexOf: uncurryThis(Array.prototype.lastIndexOf),
+ArrayPrototypePop: uncurryThis(Array.prototype.pop),
+ArrayPrototypePush: uncurryThis(Array.prototype.push),
+ArrayPrototypePushApply: UncurryThisStaticApply<typeof Array.prototype.push>
+ArrayPrototypeReverse: uncurryThis(Array.prototype.reverse),
+```
+…and so on, where `uncurryThis` is `Function.bind.bind(Function.call)`
+(also called [“call-binding”][call-bind]).
+
+[call-bind]: https://npmjs.com/call-bind
 
-<tbody>
-<tr>
-<td>
+In other words, Node.js must **wrap** every global intrinsic method
```

Comment:
```suggestion
In other words, Node.js must **wrap** every `this` sensitive intrinsic method
```

some of the things we wrap are not globals and we don't wrap every method, some static methods are fine